src/components/NavBar/utils.tsx

```diff
@@ -0,0 +1,71 @@
+import { chain, map, startCase, camelCase, mapKeys } from 'lodash';
+
+import appJSON from '../../config/prism.json';
+import layersJSON from '../../config/layers.json';
+import baseline from '../images/icon_basemap.png';
+import climate from '../images/icon_climate.png';
+import impact from '../images/icon_impact.png';
+
+const icons: { [key: string]: string } = {
```

Comment:
This is going to be config-specific (some countries might have different options). I think the `src/components/Navbar/**/*` should be all generic - anything that's deployment specific should go in `src/config`

Reply:
@bencpeters 

`icons` represent all icons. So from the `src/config/prism.json` at `categories` we can determine which one to use in the navbar.

Another way?

Reply:
Ah, I see what you're saying. This is probably a fine way to do it then.